server/isomorphic-routing/index.js

```diff
@@ -20,22 +20,23 @@ export function serverRouter( expressApp, setUpRoute, section ) {
 
 function combineMiddlewares( ...middlewares ) {
 	return function( req, res, next ) {
-		req.context = getEnhancedContext( req );
+		req.context = getEnhancedContext( req, res );
 		applyMiddlewares( req.context, ...middlewares, () => {
 			next();
 		} );
 	};
 }
 
 // TODO: Maybe merge into getDefaultContext().
-function getEnhancedContext( req ) {
+function getEnhancedContext( req, res ) {
 	return Object.assign( {}, req.context, {
 		isLoggedIn: req.cookies.wordpress_logged_in,
 		isServerSide: true,
 		path: req.url,
 		pathname: req.path,
 		params: req.params,
-		query: req.query
+		query: req.query,
+		res
```

Comment:
Is `res` the only non-isomorphic prop here, and does that matter? Could it go inside a `server` object?